### DISCUSSION TITLE
Adding a simple border to author image

### DISCUSSION THREAD

**Discussion by adamsthws** [chrede88/L1nkr#166] [Ideas]
I'd really love to be able to add a simple solid border to the author image. Please could you give me a clue as to how you might do that (with custom css file)? 

Or, possibly consider this a feature request - Allow people to add a simple border by setting something like `border=true` in thier config file?

Kind regards, Adam

**Comment by chrede88** [comment]
It might be simpler if I add an option like you suggested. In the meantime I'll sketch out the process of doing it yourself below:

Overwrite this partial: https://github.com/chrede88/L1nkr/blob/main/layouts/partials/header.html#L9
You'll have to define a new css class `author-image` (or whatever), add this class to L9. This class should include the translated classes from Tailwinds corresponding classes: `border-2 border-solid border-slate-400 dark:border-slate-50`.

These additions are essentially what has been added to the emoji overlapping the author image (L11).

**Reply by adamsthws** [reply]
Splendid, that has worked fabulously for me. Thankyou kindly!
